op-service/sources/receipts.go

```diff
@@ -138,6 +138,20 @@ var RPCProviderKinds = []RPCProviderKind{
 	RPCKindBasic,
 	RPCKindAny,
 	RPCKindStandard,
+}
+
+// Copy of RPCProviderKinds with RethDB added to all RethDB to be used but to hide it from the flags
+var validRPCProviderKinds = []RPCProviderKind{
+	RPCKindAlchemy,
+	RPCKindQuickNode,
+	RPCKindInfura,
+	RPCKindParity,
+	RPCKindNethermind,
+	RPCKindDebugGeth,
+	RPCKindErigon,
+	RPCKindBasic,
+	RPCKindAny,
+	RPCKindStandard,
 	RPCKindRethDB,
 }
```

Comment:
could to something like
```suggestion
var validRPCProviderKinds = func() []RPCProviderKind{
    return append(RPCProviderKinds, RPCKindRethDB)
}()
```
to avoid duplication.